src/views/about.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import ReactMarkdown from "react-markdown";
+import Header from "../components/header";
+import gfm from "remark-gfm";
+
+const changelogs = `
+# Changelogs
+- this is for testing purposes
+- try to delocalize this markdown
+`;
+
+export const About = () => {
+    return (
+        <div className="about-page">
```

Comment:
Since the About page details are just regular HTML content, we can keep that as a markdown file as well. Can you try whether we can save a markdown file in `public` and fetch and render it from there?

This will make editing the file easier.